### PR TITLE
docs(ee-license) Clarify and update license and Bintray doc

### PR DESCRIPTION
**Pull request by lena-larionova**
Based on https://konghq.atlassian.net/browse/DOCS-1252.

Trying to settle some of the confusion about where to find your license. The main culprit was the mention of Bintray and licenses with no context in the installation steps (people tend to ignore the prereqs). 

* Turned license prereq into an include file that's now reused for every installation doc.
* Updated "Accessing your License" with a prereq section and a link to trial/demo info.
* Updated trial section of the Introduction to Kong Enterprise topic.
* Some cleanup to installation docs - typos, formatting of the license and Bintray sections, turning URLs into actual links, turning notes into note-formatted content. 

Will backport into 1.5.x (and maybe into 1.3.x) once phrasing is reviewed and finalized.

Preview:
* [Introduction > Try Kong Enterprise](https://deploy-preview-2376--kongdocs.netlify.app/enterprise/2.1.x/introduction/)
* [Access your License](https://deploy-preview-2376--kongdocs.netlify.app/enterprise/2.1.x/deployment/access-license/) - see note and prereqs at top of page.
* Installation > [Ubuntu](https://deploy-preview-2376--kongdocs.netlify.app/enterprise/2.1.x/deployment/installation/ubuntu/) - see prereqs. Same changes in CentOS topic.
* Installation > [Amazon Linux 2](https://deploy-preview-2376--kongdocs.netlify.app/enterprise/2.1.x/deployment/installation/amazon-linux-2/) - see prereqs and links back to prereqs in step 1. Same changes in Amazon Linux 1 and Red Hat
* Installation > [Kubernetes](https://deploy-preview-2376--kongdocs.netlify.app/enterprise/2.1.x/kong-for-kubernetes/install-on-kubernetes/) - See prereqs and step 2. Same changes in other kubernetes topic.

### CODE REVIEW

File: app/_includes/md/2.1.x/bintray-and-license.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+when you begin a **paid Kong Enterprise subscription**.
+  * You will need your **username**, account **password**, account **API Key**,
+    and Kong Enterprise **license file**. For example:
+    * **Bintray username**: `john-company@kong`
+    * **Bintray password**: `12345678`
+    * **Bintray API key**: `12234e314356291a2b11058591bba195830`
+  * To find the API Key, go to [https://bintray.com/profile/edit](https://bintray.com/profile/edit) and select **API Key**.
+  * To find the Kong Enterprise license JSON file, log into your Bintray
+  account and go to `https://bintray.com/kong/<YOUR_REPO_NAME>/license#files`.
+ For more detail, see [Accessing Your License](/enterprise/latest/deployment/access-license).
```
Comment (tlblessing) [inline app/_includes/md/2.1.x/bintray-and-license.md:12]:
```suggestion
 For more details, see [Accessing Your License](/enterprise/latest/deployment/access-license).
```

File: app/enterprise/2.1.x/deployment/access-license.md [hunk 2]
```diff
@@ -4,32 +4,48 @@ title: Access Your Kong Enterprise License
...
+contact your sales representative.
+
+## Accessing your license file through a browser
+
+1. Log into [https://bintray.com/](https://bintray.com/login?forwardedFrom=%2Fkong%2F).
 If you are unaware of your login credentials, reach out to your CSE and they'll
 be able to assist you.
 
-You will notice that along with Kong Enterprise, there is a new
+2. You will notice that along with Kong Enterprise, there is a new
```
Comment (tlblessing) [inline app/enterprise/2.1.x/deployment/access-license.md:29]:
```suggestion
2. Notice that along with Kong Enterprise, there is a new
```

File: app/enterprise/2.1.x/deployment/access-license.md [hunk 3]
```diff
@@ -4,32 +4,48 @@ title: Access Your Kong Enterprise License
...
+    ![bintray-license](/assets/images/docs/ee/access-bintray-license.png)
 
-Click into the **Files** section
+4. Click into the **Files** section
 
-![bintray-license-files](/assets/images/docs/ee/access-bintray-license-files.png)
+    ![bintray-license-files](/assets/images/docs/ee/access-bintray-license-files.png)
 
-Click any file you would like to download.
+5. Click any file you would like to download.
```
Comment (tlblessing) [inline app/enterprise/2.1.x/deployment/access-license.md:40]:
```suggestion
5. Click any file you want to download.
```

File: app/enterprise/2.1.x/deployment/access-license.md [hunk 4]
```diff
@@ -4,32 +4,48 @@ title: Access Your Kong Enterprise License
...
 
 ## Programmatically accessing your license file
 
 For programmatic access you'll need 3 pieces of information:
 
- - username (was provided by email by your CSE)
- - repository name (the Bintray repository, visible in the GUI, usually named after your company name)
- - the Bintray API key for the account (see below)
+ - Username (was provided by email by your CSE)
```
Comment (tlblessing) [inline app/enterprise/2.1.x/deployment/access-license.md:46]:
```suggestion
 - Username (was provided through email by your CSE)
```

File: app/enterprise/2.1.x/deployment/installation/amazon-linux-2.md [hunk 5]
```diff
@@ -42,52 +35,60 @@ Log in to [Bintray](http://bintray.com). Your Kong Sales or Support contact will
...
-4. From the Kong version detail page, select the **Files** tab and click the distribution folder.
-5. Save the RPM file available. For example, `kong-enterprise-edition-{{page.kong_latest.version}}.aws.rpm`.
-6. Copy the RPM file to your home directory on the Amazon Linux 2 system. You may use a command like:
+1. Log in to [Bintray](http://bintray.com) using your Kong credentials. See [prerequisites](#prerequisites)
+for information on how to get access.
+2. Go to: [https://bintray.com/kong/kong-enterprise-edition-aws](https://bintray.com/kong/kong-enterprise-edition-aws).
+3. Select the `aws` folder. Kong Enterprise versions are listed in reverse chronological order.
+4. Select the latest Kong version from the list.
+5. From the Kong version detail page, select the **Files** tab and click the distribution folder.
+6. Save the RPM file available. For example, `kong-enterprise-edition-{{page.kong_latest.version}}.aws.rpm`.
```
Comment (tlblessing) [inline app/enterprise/2.1.x/deployment/installation/amazon-linux-2.md:44]:
```suggestion
6. Save the available RPM file. For example, `kong-enterprise-edition-{{page.kong_latest.version}}.aws.rpm`.
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
Suggested edits. Any comments apply globally to the content. I didn't mark every occurrence.

**Review by GayleNeumann — APPROVED** [review]
LGTM

**Review by tlblessing — APPROVED** [review]
LGTM

**Pull request merged**